album.go

```diff
@@ -11,9 +14,34 @@ type Album struct {
 	Images               []Image   `json:"images"`
 	Label                string    `json:"label"`
 	Popularity           int       `json:"popularity"`
-	ReleaseDate          time.Time `json:"release_date"`
+	ReleaseDate          string    `json:"release_date"`
 	ReleaseDatePrecision string    `json:"release_date_precision"`
 	TotalTracks          int       `json:"total_tracks"`
 	Tracks               TrackPage `json:"tracks"`
 	Name                 string    `json:"name"`
 }
+
+// SearchAlbum returns a AlbumPaging object that includes a number of albums as per the limit set
+func (a *API) SearchAlbum(q, searchType string, limit int) (*PagingAlbums, error) {
+	v := url.Values{}
+	v.Add("q", q)
+	v.Add("type", searchType)
+	v.Add("limit", strconv.Itoa(limit))
+
+	pagingAlbums := new(PagingAlbums)
+	err := a.get("v1", "/search", v, pagingAlbums)
+
+	return pagingAlbums, err
+}
+
+func (a *API) GetAlbum(name string) (*Album, error) {
+
+	pagingAlbums, err := a.SearchAlbum(name, "album", 1)
+	if err != nil {
+		return nil, err
+	}
+
+	album := pagingAlbums.Albums.Items
+
+	return album[0], nil
+}
```

Comment:
I'd really only like to add functions to this `API` that represent endpoints, so let's delete this. This is a convenience function, so it should be added similarly to how this function was added in the CLI: https://github.com/brianstrauch/spotify-cli/blob/def54fb1e6b3ae84292c4ea73d8d32ea00d336f4/internal/common.go#L70-L76.
```suggestion
```

Reply:
Let's make `SearchTrack()` and `SearchAlbum()` convenience methods in the CLI (the first already exists; it just needs to be renamed). Then, it would be really slick if we extracted the `SearchPlaylist()` functionality into the `common.go` file as well, right beneath the other two functions.